### PR TITLE
Remove teams checks.

### PR DESCRIPTION
**Pull request by anoto-moniz**
With all customers on accounts V3, we can remove all the checks for it, and capabilities that can't be used with it.

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Maintenance (non-breaking change to assist developers)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [x] I have bumped the version in __version\__.py

### CODE REVIEW

**Comment by anoto-moniz** [comment]
Holding this as a draft until the nexgen-devkit updates are merged, and I can run the full suite against this branch. I'll post a link to the job once it's kicked off.

**Comment by anoto-moniz** [comment]
https://jenkins.corp.citrine.io/job/nextgen-ITs/35420/

File: tests/test_citrine.py
Comment (bfolie) [inline tests/test_citrine.py:26]: Why is this line removed?

**Review by bfolie — APPROVED** [review]

File: tests/test_citrine.py
Comment (anoto-moniz) [reply tests/test_citrine.py:26]: That's the endpoint which was hit to retrieve the accounts version. Since the session no longer hits it, its presence on the mock was causing the test to fail.

**Pull request merged**
